net-fs/xwmfs/metadata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>matthias.gerstner@nefkom.net</email>
```

Comment:
needs bugzilla account

Reply:
I've synced mail addresses of my bugzilla account and the `metadata.xml`. It should match now.

Reply:
fine